src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -927,6 +1435,107 @@
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		037C3CAD2991A44B00B7EEE2 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CHIP_BUILD_TOOLS = true;
+				CHIP_ROOT = "$(PROJECT_DIR)/../../..";
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_CXX_LANGUAGE_STANDARD = "c++17";
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = F6SS5YK3B3;
```

Comment:
This seems wrong.  Shouldn't it be `""` like all the other DEVELOPMENT_TEAM bits in here?